### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJEclipse_Customize New Project_Keyboard: Change in color is used to represent the focus on the controls

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#3328]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier!

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.4.1; #C+E-H; #Java tooling for IntelliJ/Eclipse;

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19

**OS Details: ​**
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS 1.4.1

**NOTE: Same issue is observed for all the check-boxes and buttons**

**Repro Steps: ​**
1) Open IntelliJ IDEA application
2) Press 'Ctrl+Alt+S' to open application settings/preferences popup
3) Tab till 'Plugins' tab and press enter
4) Search for 'Azure Toolkit for IntelliJ' in the Marketplace and press enter.
5) Tab till 'Install' button and press enter
6) Restart IntelliJ IDEA application, and tab till Tools-->Azure-->Azure Sign in
7) Sign in with any Azure subscription
8) Tab till File menu and click new-->Project
9) In the New Project dialog box, select Maven and press enter
10) Check "Create from archetype" check box and select "maven-archetype-webapp" and tab till Next and press enter
11) Specify the GroupId and ArtifactId and press Next button
12) Check if the change in the color is the only way to represent the current focus on a check boxes and buttons

**Actual Result:**
Change in color is used to represent the focus on the controls

**Expected Result:**
Change in color should not be the only way to show the current focus on the control

**User Impact:**
Users with color blindness will not be able to distinguish between the focused and non-focused controls

**MAS Reference:**
[MAS 1.4.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={9f1f0271-42a1-45b1-be00-31473ba2e579})

[Bug_5_IntelliJ_Customize_Use of Color.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451212/Bug_5_IntelliJ_Customize_Use.of.Color.pptx)
![Bug_5_IntelliJ_Deploy to Azure_Use of Color](https://user-images.githubusercontent.com/41108985/62204661-4d597a80-b3ab-11e9-895a-f81027d5b433.png)

**Comment by Flanker32** [comment]
The style and behavior of a focused component is controlled by IntelliJ, which is out of the scope of out Toolkit, we have made an issue in jetbrains tacking [here](https://youtrack.jetbrains.com/issue/IDEA-221910)

**Comment by Flanker32** [comment]
- Importance of experience: Normal, users may want to switch between components, and it could be confused if the focus change is not obviously.
- Quality of experience: Low, IntelliJ provides a high contrast color theme, which shows the focus change with high contrast color, so that users with color blindness should be able to distinguish between the focused and non-focused controls
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by Flanker32** [comment]
@soumyatippapuram Please have a review.

**Comment by soumyatippapuram** [comment]
@Flanker32 Please provide the link to your AAT. This is required to follow Exclusion process. For more details, please refer : https://microsoft.sharepoint.com/teams/ceacc/SitePages/External%20Bug%20Process.aspx

**Comment by Flanker32** [comment]
Linked in [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), @soumyatippapuram  please help us verify it, thanks!

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749758
